service-war/src/main/java/com/redshiftsoft/tesla/web/mvc/changelog/ChangeLogController.java

```diff
@@ -103,11 +103,24 @@ public PageDTO<ChangeLogDTO> page(@RequestParam(required = false)
 
     @PreAuthorize("hasAnyRole('editor')")
     @Transactional
-    @RequestMapping(method = RequestMethod.POST, value = "/changes/delete")
+    @RequestMapping(method = RequestMethod.GET, value = "/changes/delete/{changeId}")
```

Comment:
I thought we were moving in the opposite direction? Why did this delete endpoint change from POST to GET?

Reply:
Dang it, good catch. I think I changed it to POST and back separately in this commit while testing changes, but I thought the rebase would catch this.